scarb/src/ops/resolve.rs

```diff
@@ -377,7 +377,12 @@ fn cairo_compilation_unit_for_target(
         components.push(CompilationUnitComponent::try_new(
             member.clone(),
             vec![target],
-            no_test_cfg_set,
+            get_cfg_with_features(
+                no_test_cfg_set.unwrap_or(cfg_set.clone()),
```

Comment:
It cannot use `cfg_set` as default - it can only be used in the main compilation unit. Please use an empty set instead.

Reply:
Sure 🫡  [b26e4b4](https://github.com/software-mansion/scarb/pull/1646/commits/b26e4b40f323e5680a933eb52051d6c990964164)